test/routes/src/server/routes/t_github.js

```diff
@@ -503,7 +654,7 @@ const hasMessage = (expected) => {
     if (typeof actual.body.payload === 'undefined'
         || typeof actual.body.payload.code === 'undefined'
         || actual.body.payload.code !== expected) {
-      throw new Error('Response does not have payload with code ' + expected);
+      throw new Error(`Response does not have payload with code ${expected} instead of ${actual.body.payload.code}`);
```

Comment:
It was annoying and very not helpful that it only showed expected value and not actual.

Possibly the same code `hasStatus` helper is in other test files too, so would be worth to update them all (or move to helper or extend `expect`).